chain/x/zoracle/internal/types/result.go

```diff
@@ -34,10 +34,11 @@ func NewResult(
 	}
 }
 
-// DecodeResult is a helper function for decoding bytes to Result.
-func DecodeResult(b []byte) (Result, error) {
+// DecodeResult is a helper function for decoding bytes to Result. The function panics if the given
```

Comment:
Must!!!

Reply:
Right. I misunderstood your comment earliler. Thanks for reminding again.